layers/+tools/google-calendar/packages.el

```diff
@@ -0,0 +1,83 @@
+;;; packages.el --- google-calendar layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Markus Johansson markus.johansson@me.com
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+(defconst google-calendar-packages
+  '(org-gcal
+    calfw
+    alert
+    f))
+
+(defun google-calendar/init-org-gcal ()
+  "Initializes org-gcal and adds keybindings for it's exposed functions"
+  (use-package org-gcal
+    :init
+    (progn
+      (spacemacs/set-leader-keys
+        "agr" 'org-gcal-refresh-token
+        "ags" 'org-gcal-sync
+        "agf" 'org-gcal-fetch))
+    :config
+    (setq org-gcal-down-days 365)   ;; Set org-gcal to download events a year in advance
+    (add-hook 'after-init-hook 'org-gcal-fetch)
+    (add-hook 'kill-emacs-hook 'org-gcal-sync)
+    (add-hook 'org-capture-after-finalize-hook 'google-calendar/sync-cal-after-capture)
+    (run-with-idle-timer 600 t 'google-calendar/org-gcal-update)))
+
+(defun google-calendar/init-calfw ()
+  "Initialize calfw"
+
+  (use-package calfw
+    :init
+    (evil-set-initial-state 'cfw:calendar-mode 'normal)
+    (defvar google-calendar-agenda-view "a"
+      "Key for opening the current week or day view in org-agenda.")
+
+    (defvar wconf nil
+      "Current window config")
+
+    (defcustom calfw-restore-windows-after-quit 't
+      "Non-nil means restore window configuration upon exiting calfw calendar view.
+The window configuration is stored when entering calendar view.
+When the view is exited and this option is set the previous layout is restored."
+      :type 'boolean)
+
+    :config
+    (evil-make-overriding-map cfw:calendar-mode-map)
+    (define-key cfw:calendar-mode-map "N" 'cfw:navi-next-month-command)
+    (define-key cfw:calendar-mode-map "P" 'cfw:navi-previous-month-command)
+    (define-key cfw:calendar-mode-map "c" 'cfw:org-capture)
+    (define-key cfw:calendar-mode-map "v" 'cfw:org-open-agenda-day))
+
+  (use-package calfw-org
+    :init
+    (spacemacs/set-leader-keys
+      "agc" 'google-calendar/calfw-view)
+    (spacemacs/declare-prefix "agc" "open-org-calendar")
+
+    :config
+    (define-key cfw:org-schedule-map "q" 'google-calendar/calfw-restore-windows)
+
+    :commands
+    (cfw:open-org-calendar)))
+
+(defun google-calendar/init-alert ()
+  "Initialize alert"
+  (use-package alert
+    :init
+    (setq alert-default-style 'notifications)))
```

Comment:
Is it required to set it? Since this is platform dependant it'd be better to let user select notification style(or we detect and set up a sane one). 
Also there are other packages/layers(slace layer, mu4e-alert) which are setting `alert-default-style` and I think it's time to consolidate effort to setup `alert` properly.